tests/Microsoft.Identity.Test.Unit/PublicApiTests/ExtensiblityTests.cs

```diff
@@ -218,6 +218,157 @@ public async Task ValidateAppTokenProviderAsync()
             }
         }
 
+        [TestMethod]
+        public async Task ValidateAdditionalCacheParametersAreStored()
```

Comment:
can you add some null and empty cache parameters checks?

Reply:
yes